mindsdb/integrations/handlers/web_handler/web_handler.py

```diff
@@ -82,32 +75,10 @@ def get_columns(self):
 
 
 class WebHandler(APIHandler):
```

Comment:
The `APIHandler` still requires a `check_connection()` method from what I can understand. The command executor still looks for it [here](https://github.com/mindsdb/mindsdb/blob/e6867f1ef42a0ff55e77f46c3ff3c44a8c96e825/mindsdb/api/executor/command_executor.py#L1099).

I think this is the reason why I am not able to execute a `CREATE DATABASE` command successfully,
![image](https://github.com/mindsdb/mindsdb/assets/49385643/c072e416-d99e-448f-b8b9-2cd3bbc4e157)

Reply:
Good point. I will update this

Reply:
@ZoranPandovski I've given it some thought, but I can't think of anything major that we would need to do here. I guess we just return `True`?